node/libs/roles/src/validator/testonly.rs

```diff
@@ -288,7 +288,24 @@ impl Distribution<GenesisRaw> for Standard {
             validators: rng.gen(),
             attesters: rng.gen(),
             leader_selection: rng.gen(),
+        };
+
+        // In order for the genesis to be valid, sticky/rota leaders need to be in the validator committee.
+        if let LeaderSelectionMode::Sticky(_) = genesis.leader_selection {
+            let i = rng.gen_range(0..genesis.validators.len());
+            genesis.leader_selection =
+                LeaderSelectionMode::Sticky(genesis.validators.get(i).unwrap().key.clone());
```

Comment:
I assume this caused some encoding test to fail. Please add more iterations for this test to get better coverage.